contracts/rentable-nft/state/src/lib.rs

```diff
@@ -9,7 +9,7 @@ use rentable_nft_io::*;
 
 #[gmeta::metawasm]
 pub mod metafns {
-    pub type State = IoNft;
+    pub type State = IoNFT;
```

Comment:
It's okay for CamelCase to use such acronyms. `Io` should be `IO`, too, if all letters are in upper case.